tasks/remove-relayer.ts

```diff
@@ -1,15 +1,22 @@
 import { task } from "hardhat/config";
+import { getDeployOptions } from "./args/deployOptions";
 
 task("remove-relayer", "Remove relayer with given address")
   .addParam("address", "Address of the relayer to be removed")
+  .addOptionalParam("price", "Gas price in gwei for this transaction", undefined)
   .setAction(async (taskArgs, hre) => {
     const address = taskArgs.address;
 
     const [adminWallet] = await hre.ethers.getSigners();
     const fs = require("fs");
-    const config = JSON.parse(fs.readFileSync("setup.config.json", "utf8"));
+    const filename = "setup.config.json";
+    const config = JSON.parse(fs.readFileSync(filename, "utf8"));
     const bridgeAddress = config["bridge"];
     const bridgeContractFactory = await hre.ethers.getContractFactory("Bridge");
     const bridge = bridgeContractFactory.attach(bridgeAddress).connect(adminWallet);
-    await bridge.removeRelayer(address);
+    await bridge.removeRelayer(address, getDeployOptions(taskArgs));
```

Comment:
In other files, there's a log line after the operation. Can we add one here, as well?